src/lib/components/CourseLinks/index.js

```diff
@@ -43,6 +43,22 @@ class CourseLinks extends HTMLElement {
     slot.addEventListener('slotchange', this.onSlotChange);
   }
 
+  /**
+   * Get the user's overall course progress from localstorage.
+   * This returns an object where each string key represents a different course.
+   * Example: {css: {pages: ['intro', ...], percent: 10}, pwa: {...}}
+   * @returns {Object.<string, Course>}
```

Comment:
nit: TS writes this like `{{[courseName: string]: Course}}`

Reply:
yeah I was trying to look up the TS syntax because I knew it was something like that. The TS JSDocs had this example
![image](https://user-images.githubusercontent.com/1066253/129284649-2a6b0eee-054f-4307-ac51-079125973f8a.png)

I'll switch it.

Reply:
Hit a bug. Since the script is inlined into the page, nunjucks is interpreting the {{ }} as templating syntax. I'm going to leave it with the Object<> style